libs/sdk-ui-ext/src/internal/components/configurationControls/DataPointsControl.tsx

```diff
@@ -0,0 +1,45 @@
+// (C) 2019 GoodData Corporation
+import React from "react";
+import { WrappedComponentProps, injectIntl } from "react-intl";
+import get from "lodash/get";
+import DropdownControl from "./DropdownControl";
+
+import { dataPointsDropdownLabels } from "../../constants/dropdowns";
+import { getTranslatedDropdownItems } from "../../utils/translations";
+import { IVisualizationProperties } from "../../interfaces/Visualization";
+
+export interface IDataPointsControlProps {
+    pushData: (data: any) => any;
+    properties: IVisualizationProperties;
+    isDisabled: boolean;
+    showDisabledMessage?: boolean;
+    defaultValue?: string | boolean;
+}
+
+class DataPointsControl extends React.Component<IDataPointsControlProps & WrappedComponentProps> {
+    public static defaultProps = {
+        defaultValue: "auto",
+        showDisabledMessage: false,
+    };
+    public render() {
+        const { pushData, properties, intl, isDisabled, showDisabledMessage, defaultValue } = this.props;
+        const dataPoints = get(properties, "controls.dataPoints.visible", defaultValue);
+
+        return (
+            <div className="s-data-labels-config">
```

Comment:
use unique s class. This one is already used ;)

Reply:
Right, I just copied this from data labels ctrl :-)